app/assets/javascripts/stories/views/StoriesEditView.js

```diff
@@ -141,6 +145,43 @@ define([
       this.render();
     },
 
+    videoInput: function(e) {
+      this._addVideoThumbnail($(e.target).val());
+    },
+
+    _getVideoID: function(url) {
+      // template: http://img.youtube.com/vi/<video-id-here>/default.jpg
+      // a Youtube video ID has a 11 characters legngth
+      return 'http://img.youtube.com/vi/' + url.split('v=')[1].substring(0, 11) + '/default.jpg';
+    },
+
+    _addVideoThumbnail: function(url) {
+      var vidID  = this._getVideoID(url),
+          $thumb = $('#videothumbnail');
+      if ($thumb.length > 0) {
+        $thumb.find('.inner_box').css('background-image','url('+ vidID +')')
+      } else {
+        $('.thumbnails').append('<li class="sortable thumbnail" draggable="true" id="videothumbnail"><div class="inner_box" style=" background-image: url('+ vidID +');"></div><a href="#" class="destroy"><svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#shape-close"></use></svg></a></li>');
```

Comment:
Maybe, we can use a handlebars for these html templates. As I see, we are using it too in line 239.